shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/general/postgresql.xml

```diff
@@ -56,19 +56,11 @@
     
     <full-insert-order>
         INSERT INTO
-        test.t_order(id,order_id,user_id,status,t_int2,t_numeric,t_bool,t_char,t_varchar,t_float,t_double,t_timestmap,t_timestamptz)
+        test.t_order_copy(id,order_id,user_id,status,t_int2,t_numeric,t_bool,t_char,t_varchar,t_float,t_double,t_timestmap,t_timestamptz)
         VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?)
     </full-insert-order>
     
     <full-insert-order-item>
         INSERT INTO test.t_order_item(item_id,order_id,user_id,status) VALUES(?,?,?,?)
     </full-insert-order-item>
-
-    <update-table-order-status>
-        UPDATE test.t_order SET status= 'unlock'
-    </update-table-order-status>
-
-    <create-index-status>
-        CREATE INDEX "idx_user_status" ON test.t_order ( status )
-    </create-index-status>
```

Comment:
Is `CREATE INDEX` not need to test any more?

Reply:
Yes, the index is used for stop source writing, no need any more

Reply:
OK